File: ws/bonifaz/u2/WS24Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Monkey.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Christian Bonifaz, Code Creators, DCCO-ESPE
 */
public class Monkey extends Primate{

    public Monkey(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    @Override
    public void Feed(int food) {
        System.out.println("Feeding a monkey");
    }
}

File: hw/ccedeno/u2/HW18ZooInPythonJavaScript/ZooInJavaScript/ec.edu.espe.Zoo.model/Monkey.js
const Primate = require('../ec.edu.espe.Zoo.model/Primate');

class Monkey extends Primate {
  constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
    super(id, description, dateOnborn, gender, numberOfBones, cage);
  }

  feed(food) {
    console.log("Feeding a monkey");
  }
}

module.exports = Monkey;
